test/crypto_test.go

```diff
@@ -2,12 +2,14 @@ package test
 
 import (
 	"bytes"
-	"github.com/stretchr/testify/assert"
-	"github.com/wal-g/wal-g/internal"
-	"golang.org/x/crypto/openpgp"
 	"io/ioutil"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal/crypto"
+	walg_openpgp "github.com/wal-g/wal-g/internal/crypto/openpgp"
```

Comment:
Maybe it's okay.

Reply:
Fixed. Moved this to internal/crypto/openpgp.